xline-test-utils/src/bin/validation_lock_client.rs

```diff
@@ -1,7 +1,11 @@
 //! this binary is only used for the validation of lock service
 
 use clap::{Parser, Subcommand};
-use etcd_client::Client;
```

Comment:
could we remove etcd_client dependency in Cargo.toml?

Reply:
Removed